### ISSUE TITLE
A11y_.NETCore3.0(Win Forms and CoreSDK)_TreeView_Screenreader: NVDA is not announcing tree view items properly.

### ISSUE THREAD

**Issue by mspatil110** [nvaccess/nvda#10222]
[“Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-shbagl%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847575447&sdata=XQqvaO2rFh7p/7F2DmzFj2wgJSiYHGuJV73Xg8hPVbA%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:​**

Application Name: .NET Core 3.0 (WinForms and CoreSDK)

Windows Version: Windows10​

Visual Studio 2019 version:  16.1.3

**Repro Steps:​**

1.Launch Core testing project and press F5.
2. Navigate to "common controls_2: Testing the control under common control tab" and hit enter.
3. Start NVDA.

4.Verify that NVDA is announcing  tree view items properly or not.

**Actual:**

NVDA is announcing the tree view item Node0 collapsed 1 of 1 and Node3 collapsed 1 of 1.

**Note:** Narrator is announcing the tree view items properly like 
 Node0 collapsed 1 of 2 and Node3 collapsed 2 of 2.

**Expected:**

NVDA should announce the tree item like Node0 collapsed 1 of 2 and Node3 collapsed 2 of 2.

**User Impact:**

Screen reader only users will face difficulty to understand the tree view items properly.

**Recommendations:**
Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:**

MAS 1.3.1:

https://microsoft.sharepoint.com/:w:/r/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx?d=w54f28d1fa2d14dcd84e15c9b87e8aba4&csf=1&e=s3uErN

**Attachment:**
![956724_A11y_ NETCore3 0(Win Forms and CoreSDK)_TreeView_Screenreader](https://user-images.githubusercontent.com/51433293/64959019-eab23480-d8ad-11e9-9294-66d857ce15e8.png)

**Comment by leonardder** [comment]
This most likely has the same underlying issue as #7041.

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
